### PR TITLE
Re-apply "Make the .deploy rule resolve the dependencies" (#35)"

### PR DESCRIPTION
**Pull request by kchodorow**
CC @regisd

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,11 +95,16 @@ java_war(
...
-You can then build the application with `bazel build //hello_app:myapp` and
-run in it a development server with `bazel run //hello_app:myapp`. This will
-bind a test server on port 8080. If you wish to select another port,
+You can then build the application with `bazel build //hello_app:myapp`.
+
+### Run on a local server
+
+You can run it in a development server with `bazel run //hello_app:myapp`.
+This will bind a test server on port 8080. If you wish to select another port,
 simply append the `--port=12345` to the command-line.
```
Comment (pmbethe09) [inline README.md]: mention '-- --port=12345' ?  The bazel '--' cutoff?

**Comment by pmbethe09** [comment]
LGTM

File: README.md (hunk 1 above)
Comment (kchodorow) [inline README.md]: Addressed in pull request #39.

Comment (kchodorow) [inline README.md]: Addressed in pull request #39.

**Pull request merged**